packages/app/src/scenes/widgets/pages/StakingViewWidget/components/MyWallet/MyWallet.tsx

```diff
@@ -106,26 +108,31 @@ const MyWallet: FC<PropsInterface> = ({
           />
         </styled.RewardsContainer>
 
-        <styled.Title>Use Faucet</styled.Title>
-        <styled.AirdropContainer>
-          {canRequestAirdrop !== false ? (
+        {appVariables.CONTRACT_FAUCET_ADDRESS !== "" ?? (
```

Comment:
```suggestion
        {!!appVariables.CONTRACT_FAUCET_ADDRESS && (
```
This is a bit clearer to me :)
Using the double negation here means that if it's non-empty string, null or undefined, we'll render the Faucet components.